src/ui_common.h

```diff
@@ -4,16 +4,38 @@
 
 #include "ux.h"
 
+#ifdef HAVE_NBGL
+#include "nbgl_use_case.h"
+
+extern nbgl_layoutTagValue_t tagValuePair[5];
+extern nbgl_layoutTagValueList_t tagValueList;
+extern nbgl_pageInfoLongPress_t infoLongPress;
+#endif // HAVE_NBGL
+
 void ui_idle(void);  // must be implemented by each ui handler
+void ui_display_public_flow(void); 
+void ui_display_sign_flow(void); 
+void ui_confirm_selector_flow(void);
+void ui_confirm_parameter_flow(void);
+void ui_approval_celo_lock_unlock_flow(void);
+void ui_approval_celo_withdraw_flow(void);
+void ui_approval_celo_vote_revoke_flow(void);
+void ui_approval_celo_activate_flow(void);
+void ui_approval_celo_relock_flow(void);
+void ui_approval_celo_create_account_flow(void);
+void ui_approval_celo_data_warning_gateway_tx_flow(void);
+void ui_approval_celo_gateway_tx_flow(void);
+void ui_approval_celo_data_warning_tx_flow(void);
+void ui_approval_celo_tx_flow(void);
 
-unsigned int io_seproxyhal_touch_data_ok(const bagl_element_t *e);
-unsigned int io_seproxyhal_touch_data_cancel(const bagl_element_t *e);
+unsigned int io_seproxyhal_touch_data_ok(void);
```

Comment:
If you are brave enough, I thing those can be renamed...
`ui_callback_data_ok` or something like that would probably be better...